tests/input_flows.py

```diff
@@ -381,6 +387,71 @@ def input_flow_tr(self) -> BRGeneratorType:
         self.debug.press_middle()
 
 
+class InputFlowShowXpubQRCode(InputFlowBase):
+    def __init__(self, client: Client, passphrase: bool = False):
+        super().__init__(client)
+        self.passphrase = passphrase
+
+    def input_flow_tt(self) -> BRGeneratorType:
+        if self.passphrase:
+            br = yield
+            self.debug.press_yes()
+            br = yield
+            self.debug.press_yes()
+
+        br = yield
+        layout = self.debug.wait_layout()
+        if "coinjoin" in layout.title().lower():
+            self.debug.press_yes()
+            br = yield
+        elif br.code == B.UnknownDerivationPath:
```

Comment:
Doing the same action in both branches - make it an `or` statement?

Reply:
e49b023